docs/topics/certificaterotation.md

```diff
@@ -0,0 +1,52 @@
+# Cluster certificate rotation
+
+Instructions on rotating TLS CA and certificates for an AKS Engine cluster.
+
+## Prerequesites
+
+- Kubernetes version 1.8.0 or later is required
+- The etcd members MUST be in a healthy state before rotating the CA and certs.
+- The `_output` directory for your cluster deployment, including a working SSH key for accessing the master nodes and `kubeconfig`.
+
+## Preparation
+
+**CAUTION**: Before rotation, it's preferrable to back up your cluster:
+
+1. On an etcd node, take a backup of the current state.
```

Comment:
Do you need to do this on an etcd node literally? Can you do this from anywhere that has an authenticated connection to the etcd cluster?

Reply:
I don't know, the etcd backup is mostly TODO, it's not part of the cert-rotation functionality I'm adding, it's a pre-req for which we will provide a tool recommendation (@palma21 was going to look into that). These are just standard instructions I got from the coreos website.

Reply:
Yeah, the user can do it without directly hopping into the node, but that is one option.

Suggest we generalize it as
Backup the etcd cluster data